x-pack/plugins/ml/server/models/job_service/new_job/categorization/validation_results.ts

```diff
@@ -123,15 +123,18 @@ export class ValidationResults {
   public createNullValueResult(examples: Array<string | null | undefined>) {
     const nullCount = examples.filter((e) => e === null).length;
 
-    if (nullCount / examples.length >= NULL_COUNT_PERCENT_LIMIT) {
-      this._results.push({
-        id: VALIDATION_RESULT.NULL_VALUES,
-        valid: CATEGORY_EXAMPLES_VALIDATION_STATUS.PARTIALLY_VALID,
-        message: i18n.translate('xpack.ml.models.jobService.categorization.messages.nullValues', {
-          defaultMessage: 'More than {percent}% of field values are null.',
-          values: { percent: NULL_COUNT_PERCENT_LIMIT * 100 },
-        }),
-      });
+    // add check so message won't be added if all values are null
```

Comment:
if 100% of fields are `null`, why do we not want to report this?

Reply:
Sorry I should have been clearer in the comment. 

Currently, if all values are null, the message we are giving back is: `'No examples for this field could be found. Please ensure the selected date range contains data.'` 

The added check here ensures we only give that one message back instead of both `'More than 75% of field values are null.'` **and** `'No examples for this field could be found.'`

Reply:
ok makes sense.
It think it might worth adding something to this comment for future readers. Something like:
```
If all values are null, VALIDATION_RESULT.NO_EXAMPLES will be raised. So we don't need to display this warning as well
```

Reply:
Thanks. Updated the comment to be more explicit here. https://github.com/elastic/kibana/pull/78700/commits/219c66171ac312c2cdbe6bca43cb42ef9a72d515